frontend/encore/simple-example.rst

```diff
@@ -128,7 +128,7 @@ Great! Use ``require()`` to import ``jquery`` and ``greet.js``:
     // ./ (or ../) means to look for a local file
     var greet = require('./greet');
 
-    $(document).ready(function()) {
+    $(document).ready(function() {
         $('h1').html(greet());
```

Comment:
`greet(...)` function needs an argument to complete the job https://github.com/symfony/symfony-docs/pull/8032/files#diff-4a37b5376878b65036d1bb0fa49047f7R132